packages/leaflet/leaflet.0.1/opam

```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings for the Leaflet JavaScript library"
+description:
+  "leaflet is an OCaml bindings library for the Leaflet JavaScript library."
+maintainer: ["swrup <swrup@protonmail.com>"]
+authors: [
+  "pukkamustard <pukkamustard@posteo.net>"
+  "swrup <swrup@protonmail.com>"
+  "Léo Andrès <contact@ndrs.fr>"
+]
+license: "BSD-2-Clause"
+tags: ["leaflet" "javascript" "bindings" "interactive" "map" "openstreetmap"]
+homepage: "https://git.zapashcanon.fr/swrup/leaflet"
+doc: "TODO/leaflet"
+bug-reports: "https://git.zapashcanon.fr/swrup/leaflet/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "brr"
+  "ocaml" {>= "4.08"}
```

Comment:
```suggestion
  "js_of_ocaml"
  "js_of_ocaml-ppx"
  "ocaml" {>= "4.08"}
```

I know you use brr which transitively installs it, but since you link and use explicitly js_of_ocaml and its rewriter, it should be listed among the dependencies